test/data/mock/index.ts

```diff
@@ -1,2 +1,3 @@
 export * as Context from './context';
 export * as Payload from './payload';
+export * as StateStore from './statestore';
```

Comment:
[suggestion] It would be better if we could merge all data in this file into `payload.ts`.